src/footprints/index.js

```diff
@@ -11,5 +11,9 @@ module.exports = {
     rgb: require('./rgb'),
     jstph: require('./jstph'),
     pad: require('./pad'),
-    rotary: require('./rotary')
+    rotary: require('./rotary'),
+	// Fresh prints
```

Comment:
Fresh prints... of Bel Air? :smile: Anyhow, this comment is probably not needed (and we could maybe even alphabetize the list for good measure).